### PR TITLE
kde-plasma/qqc2-breeze-style: new package, add 9999

### PR DESCRIPTION
**Pull request by parona-source**
Noticed talk on irc, so here is the package. @redstrate

### CODE REVIEW

File: kde-plasma/qqc2-breeze-style/metadata.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="project">
+		<email>kde@gentoo.org</email>
+		<name>Gentoo KDE Project</name>
+	</maintainer>
+	<upstream>
+		<bugs-to>https://bugs.kde.org/</bugs-to>
```
Comment (redstrate) [inline kde-plasma/qqc2-breeze-style/metadata.xml:9]: I think we are using the Invent bug tracker, or at least not Bugzilla. Or at least it didn't have a product the last time I checked...

**Comment by a17r** [comment]
Do we depend on this from anywhere?

**Comment by redstrate** [comment]
> Do we depend on this from anywhere?

Not yet, but plasma-integration has begun to start enforcing this for QGuiApplications if they don't set one specifically so some applications (a very small portion of them, to be frank) will start using this style.

**Review by a17r — COMMENTED** [review]
@redstrate: this seems like an upstream leftover since it does not exist in Qt6:
```
-- The following OPTIONAL packages have not been found:

 * Qt6X11Extras (required version >= 6.6.0)
```

File: kde-plasma/qqc2-breeze-style/metadata.xml (hunk 1 above)
Comment (a17r) [reply kde-plasma/qqc2-breeze-style/metadata.xml:9]: That would normally be reserved for dev bugs, not user bugs. Maybe something that needs to be requested from KDE infra.

**Comment by parona-source** [comment]
https://doc.qt.io/qt-6/extras-changes-qt6.html#changes-to-qt-x11-extras

"The [Qt X11 Extras](https://doc.qt.io/qt-5/qtx11extras-index.html) module was removed in Qt 6."

Yep.

**Review by a17r — APPROVED** [review]
Thanks!

**Pull request closed without merging**